src/app.js

```diff
@@ -1 +1,25 @@
 'use strict';
+
+const readline = require('readline');
+const { getRandomNumber } = require('./getRandomNumber');
+const { counterResults } = require('./counterResults');
+
+const gameInterface = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+function cowsAndBuls() {
+  gameInterface.question('Write four digit number', (num) => {
+    const result = counterResults(num, getRandomNumber());
```

Comment:
The number generated by getRandomNumber() should be stored outside the question callback and reused during the game.